Dbal/RegexSchemaAssetFilter.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Doctrine\Bundle\DoctrineBundle\Dbal;
+
+use Doctrine\DBAL\Schema\AbstractAsset;
+
+class RegexSchemaAssetFilter implements SchemaAssetFilterInterface
```

Comment:
Shouldn't this be part of DBAL instead ?

Reply:
Possibly. But, would they want this there? They don't actually need this class, and so they would have this class just to support other people composition class-based callbacks.

Reply:
Kinda agree that this should be provided by DBAL, but then they only made it `callable` so there's nothing really we can do. For now we can have the classes here, maybe we can revisit this in a later DBAL release.